### PR TITLE
Add github flavored markdown table implementation.

### PR DESCRIPTION
**Pull request by bcho**
Note: this implementation will disable the usage of title.

### CODE REVIEW

File: terminaltables/tables.py [hunk 1]
```diff
@@ -92,3 +92,66 @@ class DoubleTable(WindowsTableDouble):
...
+class GithubFlavoredMarkdownTable(BaseTable):
+    """Github flavored markdown table.
+
+    https://help.github.com/articles/github-flavored-markdown/#tables
+    """
+
+    CHAR_VERTICAL = '|'
+    CHAR_HORIZONTAL = '-'
+
+    def __init__(self, table_data):
```
Comment (bcho) [inline terminaltables/tables.py]: @Robpol86 CI warns for this issue: `W:106, 4: __init__ method from base class 'BaseTable' is not called (super-init-not-called)`, but I think there is no need to call the super here.

Comment (Robpol86) [inline terminaltables/tables.py]: You need to call super(GithubFlavoredMarkdownTable, self).init(table_data) within your __init__().
http://stackoverflow.com/questions/576169/understanding-python-super-with-init-methods

Comment (bcho) [inline terminaltables/tables.py]: @Robpol86 ok, fixed now.

**Comment by Robpol86** [comment]
thanks!

**Pull request merged**